ui/apps/dev-server-ui/src/app/(dashboard)/stream/useEvent.ts

```diff
@@ -1,10 +1,11 @@
 import { useMemo } from 'react';
 import type { Event } from '@inngest/components/types/event';
 import {
-  baseFetchFailed,
-  baseFetchLoading,
   baseFetchSkipped,
   baseFetchSucceeded,
+  baseInitialFetchFailed,
```

Comment:
I'm not a fan of mixing dev server stuff in this PR, did you test the Dev server just in case?

Reply:
Yea, it still works. And I had to change it because they share a bit of code